docs/output.html

```diff
@@ -0,0 +1,136 @@
+<html>
```

Comment:
Should this be here? Looks like this a already generated output?

Reply:
Ah no, the output file does get committed when the release versions job is run but it sits on the scripts-pages branch which is used by GitHub pages to show it so it shouldn't be on this branch. I've deleted it.